tests/test_commander_pro.py

```diff
@@ -864,7 +864,7 @@ def test_set_color_hardware_default_start_end(commanderProDevice):
 
     assert sent[3].data[0] == 0x35
     assert sent[3].data[2] == 0x00  # start led
-    assert sent[3].data[3] == 0x01  # num leds
+    assert sent[3].data[3] == 203  # num leds
```

Comment:
Aren't there 204 LEDs between 1 and 204?  So if the comment is right ("num leds"), should this assert if the value is equal to 204?

Reply:
I may have been off by one in the initial code that set the value. I believe there was one user that had the maximum number. I asked the original reporter if they are able to confirm the off by one error. But ill make the change to fix that as well and add a test for it if there is not one already